### DISCUSSION TITLE
Fragen: COTA / FOTA / POTA....

### DISCUSSION THREAD

**Discussion by ds2k5** [wavelog/wavelog#309] [Q&A] [answered]
Hallo,
wenn man z.B: COTA macht wie kann man dann den eigenen QTH, Locator Code anpassen ?
Wenn man das QSO eintragen will.
Wie setzt man dann z.B.  **/p** zum Rufzeichen hinzu ?

**Comment by HB9HIL** [comment, marked as answer]
Hallo @ds2k5 

Wavelog unterstützt nativ die folgenden Special Activities:

IOTA
SOTA
POTA
WWFF

Für diese Aktivitäten findest du in den "Station Locations" jeweils entsprechende Felder um dort den Referenz Code einzutragen. 
Andere Aktivitäten wie COTA, FOTA, etc. welche in Wavelog nicht nativ implementiert sind kannst du in der Station Location die Felder SIG und SIG_INFO benutzen. 

Machst du Betrieb von einem neuen COTA Standort aus, legst du hierfür eine neue Station Location an (die Copy Funktion hilft enorm bei ähnlichen Locations). 
Nun trägst du im SIG Feld "COTA" und im SIG_INFO die Referenz, z.B. "BOB-001" für die Burg Falkenberg ein. 
Arbeitest du nun speziell bei COTA nach dem internationalen WCA, dann trägst du für die Burg Falkenberg in SIG "WCA" und in SIG_INFO "DL-02335" ein. 

SIG ist immer die Aktivität, bzw. die Kategorie
SIG_INFO ist die dazugehörige Referenz.

In der jeweiligen Station Location setzt du dann auch das Callsign, welches du bei dieser Aktivität verwendest. Zum Beispiel HB9HIL/P oder HB9HIL/M

Wichtig ist, dass bevor du QSO's dann für diesen Standort logst, du die Location aktiv schaltest. Das geht im Station Setup. 

Funkst du nun von einem weiteren Standort aus, mit einem anderen COTA Code, so erstellst du hierfür wieder eine neue Station Location. Das geht mit der Copy Funktion inner Sekunden. 

Klärt das deine Frage?

Vy 73 de HB9HIL

**Comment by ds2k5** [comment]
@HB9HIL 

vielen Dank für die schnelle Hilfe

>Klärt das deine Frage?

Denke ja...

**Reply by HB9HIL** [reply]
@ds2k5 gern :)
